dastardcommander/dc.py

```diff
@@ -144,18 +145,23 @@ def updateReceived(self, topic, message):
         try:
             d = json.loads(message)
         except Exception as e:
-            print("Error processing status message [topic,msg]: %s, %s" % (
+            print("Error processing status message [topic,msg]: '%s', '%s'" % (
                 topic, message))
             print("Error is: %s" % e)
             return
 
-        quietTopics = set(["TRIGGERRATE", "NUMBERWRITTEN", "ALIVE", "EXTERNALTRIGGER"])
+        quietTopics = set(["TRIGGERRATE", "NUMBERWRITTEN", "EXTERNALTRIGGER"])  # add "ALIVE"
```

Comment:
this looks like you removed ALIVE and added a comment to add ALIVE? that seems counterproductive?